### ISSUE TITLE
Ribotaper issue with container

### ISSUE THREAD

**Issue by mkatsanto** [bioconda/bioconda-recipes#48920]
Hi, I have an issue with the ribotaper container https://quay.io/repository/biocontainers/ribotaper?tab=tags . None of the versions work, while the corresponding conda package works. There seems
to be an issue with the readlink version (see below for the detailed error). Can someone have a look?
Error 

> Usage: readlink [-fnv] FILE
> 
> Display the value of a symlink
> 
>         -f      Canonicalize by following all symlinks
>         -n      Don't add newline
>         -v      Verbose
> 
> readlink: invalid option -- 'e'
> BusyBox v1.22.1 (2014-05-23 01:24:27 UTC) multi-call binary.
> 
> Usage: readlink [-fnv] FILE
> 
> Display the value of a symlink
> 
>         -f      Canonicalize by following all symlinks
>         -n      Don't add newline
>         -v      Verbose
> 
> Parameters used:
> 
> <gtf_file> 
> <genome_fasta_file(indexed)>

**Comment by corneliusroemer** [comment]
Thanks for opening the issue!

Could you possibly provide the exact command(s) you run that result in output that's not how it should be?

How do you start the container, what did you run inside it? etc

**Comment by mkatsanto** [comment]
Thanks for the reply, I am working on a linux system and the command failing is:

`singularity exec ribotaper_1.3.1a--3.sif create_annotations_files.bash             results/mus_musculus/gtf.wihout_scaffolds.gtf             results/mus_musculus/genome.wihout_scaffolds.fasta             false            false             results/mus_musculus/RiboTaper`

where the options are as suggested in the manual.
In the case of conda I activated the following env (in yaml)
```
---
channels:
  - conda-forge
  - bioconda
dependencies:
  - ribotaper=1.3.1
...

``` and then executed the same command :
`create_annotations_files.bash             results/mus_musculus/gtf.wihout_scaffolds.gtf             results/mus_musculus/genome.wihout_scaffolds.fasta             false            false             results/mus_musculus/RiboTaper`

I also executed the singularity command as part of a snakemake workflow and when I executed I made sure that the working directory was in the bindings. Also, other similar rules pointing to the same files and using singularity were executed. Do not know if this is helpful.

The line that seems to be failing in the code is here:
https://github.com/boboppie/RiboTaper/blob/2a8aa2efd131d29cb15dcf04a205e83de79d9b01/scripts/create_annotations_files.bash#L65

**Comment by corneliusroemer** [comment]
Ah great! It does indeed look like the version of readlink that the container uses (from busybox) does not support the option `-e`

Adding readlink as a run dependency might fix it

**Comment by corneliusroemer** [comment]
@mkatsanto can you test with the latest container? It should be ready in a few minutes!

**Comment by mkatsanto** [comment]
Yes will test and get back to you. Hello to you too. I just saw that as well. Haha that is random indeed.

**Comment by mkatsanto** [comment]
Is this the corrected version? 1.3.1a--h031d066_7 
It is still not working. 
An easy way to test is to do  singularity run ribotaper_1.3.1a--h031d066_7.sif
and then run the readlink -e within the prompt.

**Comment by corneliusroemer** [comment]
Ah yeah sorry my bad, of course it doesn't work - I added "readline" not "readlink" 🤦  New PR: #48946

**Comment by corneliusroemer** [comment]
@mkatsanto could you please try again? Now I actually added "readlink" via "coreutils" and not readline :D

**Comment by mkatsanto** [comment]
Sorry for the delayed answer. It is working now! Thanks! I will close the issue now.

**Comment by corneliusroemer** [comment]
Excellent! Thanks for reporting and sorry for me readline messup 😄
